src/page.rs

```diff
@@ -180,3 +180,41 @@ pub fn login(message: Option<&str>) -> Response<Body> {
         }
     }
 }
+
+pub fn register(message: Option<&str>) -> Response<Body> {
+    let page = html! {
+        : doctype::HTML;
+        html {
+            head {
+                meta(name="viewport", content="width=device-width, initial-scale=1");
+                title {
+                    : Raw("Register");
+                }
+                style { : Raw(include_str!("../assets/style.css")); }
+            }
+            body {
+                div(class="register-page") {
+                    form(method="POST", class="form") {
+                        input(type="text", name="user", placeholder="username", autofocus);
+                        input(type="password", name="pass", placeholder="password");
```

Comment:
A second `password` input for confirmation would be nice.

Reply:
only added the input for confirmation, but no checking logic is done yet